templates/plugin-gitlab.mustache

```diff
@@ -29,13 +29,13 @@ PHPunit:PHP5.3:MySQL:
   services:
     - mysql:5.6
   script:
-  - phpcs --standard=phpcs.ruleset.xml $(find . -name '*.php')
+  - phpcs --standard=phpcs.xml.dist $(find . -name '*.php')
   - phpunit --configuration phpunit.xml.dist
 
 PHPunit:PHP5.6:MySQL:
   image: tetraweb/php:5.6
   services:
     - mysql:5.6
   script:
-  - phpcs --standard=phpcs.ruleset.xml $(find . -name '*.php')
-  - phpunit --configuration phpunit.xml.dist
\ No newline at end of file
+  - phpcs --standard=phpcs.xml.dist $(find . -name '*.php')
```

Comment:
I think that the `--standard=...` should be removed in the CI configurations as well.

Otherwise, you might end up creating a customized rule set that is being used for local tests, while the CI still runs with the default tests.